### PR TITLE
Feature/activity dom functionality

### PR DESCRIPTION
**Pull request by WinstonCalhoun**
This will add the daily/weekly DOM functionality for display and updates to the Activity class methods for connection.

Next up is refactoring for cleaner looking code and well as syntax cleanup.

### CODE REVIEW

File: src/data/Activity.js [hunk 1]
```diff
@@ -2,7 +2,35 @@ class Activity {
...
+
+  returnDailySteps(userID, date) {
+    const activityEntries = this.activityData.filter(activityEntry => activityEntry.userID === userID);
+    const dailyEntry = activityEntries.find(entry => {
+      return entry.date === date ;
+    });
+    return dailyEntry.numSteps
+  };
+
+  returnWeeklySteps(userID, startDate) {
```
Comment (Devynnem) [inline src/data/Activity.js:15]: DOM is looking fresh!

File: src/scripts.js [hunk 2]
```diff
@@ -120,4 +126,23 @@ function displayWeeklySleepQuality() {
...
+function displayActivity() {
+    dailySteps.innerText = `You took ${activity.returnDailySteps(newUser.id, currentDate)} steps today!`
+    dailyMiles.innerText = `You have walked ${activity.returnMiles(newUser.id, currentDate)} miles today!`
+    dailyMinutes.innerText = `You were active for ${activity.returnMinutesActive(newUser.id, currentDate)} minutes today!`
+};
+
+function displayWeeklyStepCount() {
+    const weeklyActivityEntries = activity.returnWeeklySteps(newUser.id, currentDate); 
+    weeklyActivityEntries.forEach(entry => {
+        if(activity.returnMetStepGoal(newUser.id, entry.date)) {
```
Comment (sarahcatherine311) [inline src/scripts.js:140]: Love this so much!! Time to refactor all of the for loops! Let's go!

**Review by sarahcatherine311 — APPROVED** [review]
Looks great!!

**Review by tdmburr — COMMENTED** [review]
This looks great! Great job on the dom manip. We can move on to refactoring where necessary.

**Pull request merged**